magenta/music/musicxml_parser.py

```diff
@@ -552,43 +578,64 @@ def _fix_time_signature(self):
     """
     # Compute the fractional time signature (duration / divisions)
     # Multiply divisions by 4 because division is always parts per quarter note
-    fractional_time_signature = Fraction(self.duration,
-                                         self.state.divisions * 4)
+    numerator = self.duration
+    denominator = self.state.divisions * 4
+    fractional_time_signature = Fraction(numerator, denominator)
 
     if self.state.time_signature is None and self.time_signature is None:
       # No global time signature yet and no time signature defined
       # in this measure (no time signature or senza misura).
       # Insert the fractional time signature as the time signature
       # for this measure
-
       self.time_signature = TimeSignature(self.state)
       self.time_signature.numerator = fractional_time_signature.numerator
       self.time_signature.denominator = fractional_time_signature.denominator
       self.state.time_signature = self.time_signature
     else:
+      #print self.state.time_signature, self.time_signature
       # If no global time signature yet, set it to the
       # time signature in this measure
       if self.state.time_signature is None:
         self.state.time_signature = self.time_signature
```

Comment:
Given that you now assign time_signature to self.state.time_signature whenever you assign to self.time_signature, is this code still relevant?